src/codewhisperer/util/commonUtil.ts

```diff
@@ -61,6 +62,10 @@ export function getPrefixSuffixOverlap(firstString: string, secondString: string
     return secondString.slice(0, i)
 }
 
+export function getOptOutPreference() {
+    return globals.telemetry.telemetryEnabled ? 'OPTIN' : 'OPTOUT'
```

Comment:
I'm confused about the code organization in /codewhisperer/. 

Isn't this only for telemetry? Why doesn't it live in TelemetryHelper? https://github.com/aws/aws-toolkit-vscode/blob/8ff69bb8d10824260e03afdfdb3ae61ff27eff6d/src/codewhisperer/util/telemetryHelper.ts#L37

Reply:
yea agree, but I think not only to this PR, we do have a lot of files needed to be reorganaized

Reply:
It's not only for telemetry, the values will apply to the `GenerateCompletions` API and `SendTelemetryEvent` APIs. It's more of a helper util function to convert values.